src/core/Akka/Util/Base64Encoding.cs

```diff
@@ -24,14 +25,40 @@ public static class Base64Encoding
         /// </summary>
         /// <param name="value">TBD</param>
         /// <returns>TBD</returns>
-        public static string Base64Encode(this long value) => Base64Encode(value, new StringBuilder()).ToString();
+        public static string Base64Encode(this long value)
+        {
+            return Base64Encode(value, "");
+        }
+
+        internal static string Base64Encode(this long value, string prefix)
+        {
+            // 11 is the number of characters it takes to represent long.MaxValue
+            // so we will never need a larger size for encoding longs
+            Span<char> sb = stackalloc char[11 + prefix?.Length ?? 0];
+            var spanIndex = 0;
+            if (!string.IsNullOrWhiteSpace(prefix) && prefix.Length > 0)
+            {
+                prefix.AsSpan().CopyTo(sb);
+                spanIndex = prefix.Length;
+            }
+
+            var next = value;
+            do
+            {
+                var index = (int)(next & 63);
+                sb[spanIndex++] = Base64Chars[index];
```

Comment:
I wonder, is there any advantage at this point to changing Base64Chars to `ReadOnlyMemory`?

Reply:
I don't think so since it's basically just integer accessors